src/components/Molecules/Membership/Form/Form.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import Text from '../../../Atoms/Text/Text';
+import MoneyBuy from '../MoneyBuy/MoneyBuy';
+import {
+  onKeyPress,
+  isAmountValid,
+  getUrlParameter,
+  isInputMatchBoxValue
+} from '../../../../utils/Membership';
+import {
+  Button,
+  Copy,
+  Error,
+  FormFieldset,
+  FormWrapper,
+  Label,
+  Form,
+  MoneyBuys,
+  AmountField
+} from '../Membership.style';
+
+const Signup = ({ data: { regularGiving }, ...rest }) => {
+  // It's used to hightlight one of the money buy box when page load
+  const [isSelected, setIsSelected] = useState(true);
+  const [moneyBoxes, setMoneyBoxes] = useState({
+    box1: '',
+    box2: '',
+    box3: ''
+  });
+
+  const [errorMsg, setErrorMsg] = useState(false);
+  const [amountDonate, setAmountDonate] = useState('');
+  const [userInput, setUserInput] = useState('');
+  const [boxBorderColor, setBoxBorderColor] = useState('');
+  const [inputBorderColor, setInputBorderColor] = useState(false);
+  const [moneyBuyCopy, setMoneyBuyCopy] = useState(true);
+
+  const selectMoneyBuy = (copy, value) => {
+    setIsSelected(false);
+    if (errorMsg) setErrorMsg(false);
+    const isUserInputMatch = userInput * 1 === value * 1;
```

Comment:
`userInput === value` is the same thing, no?!

Reply:
Nope `userInput` is a string

Reply:
I mean `userInput === value` is the same as `userInput * 1 === value * 1`. No? Why multiply a string by 1?

Reply:
Sorry,  it should be like this `userInput * 1 === value`

Reply:
Could always use parseInt to make it clearer what's going on